openedx/adg/lms/registration_extension/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+`registration_extension` app customizes edX out of the box registration. Any kind of customizations
+in registration or UserProfile are made here. In case we add or remove any field(s) in registration form
+we need to update REGISTRATION_EXTRA_FIELDS setting accordingly.
+Here is the link to the guide to update the above mentioned setting,
```

Comment:
Since it is started from a new line, it is better to leave a blank line, to give it a paragraph-like look.